Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.cs

```diff
@@ -81,5 +103,155 @@ protected override void OnApplyTemplate()
 
             base.OnApplyTemplate();
         }
+
+        private void OnApplyTemplateNavView()
+        {
+            if (_navigationView is NavigationView navView && navView != null)
```

Comment:
The `&& navView != null` bit is not required, as a null `object` can never be of type `NavigationView`.

The condition is only true when `_navigationView` has been set to an instance of `NavigationView` (or extended class).

Reply:
Yup, agree, removed